### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by techwords**
Updated the introduction and added the "id" for Inputs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
 ![image](https://github.com/hiddenlayerai/hiddenlayer-model-scan-github-action/assets/9558507/dbebe8db-7e68-479d-a244-070238adf00d)
 
+For more information about GitHub Actions:
+
+- Read the [Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) page.
+- Read the [GitHub Actions Quickstart](https://docs.github.com/en/actions/quickstart) for a quick overview.
+
 ## Inputs
 
+`id` (required): Scans a model file or scans all model files in a folder. Supported values are: `scan_model` and `scan_model_folder`.
```
Comment (Ashton-Sidhu) [inline README.md:21]: This field is a Github Action field and can be anything the user wants. It is the name of the task in the overall GitHub workflow.

**Comment by Ashton-Sidhu** [comment]
Don't worry about the failing tests, looks like its a permissions issue on s3

File: README.md (hunk 1 above)
Comment (techwords) [reply README.md:21]: OK. Then I'll remove it. I thought the user needed to select one or the other.

**Review by Ashton-Sidhu — APPROVED** [review]
Thanks for doing this !

**Pull request merged**